src/status_im/data_store/core.cljs

```diff
@@ -18,3 +19,23 @@
 
 (defn change-account [address new-account? encryption-key handler]
   (data-source/change-account address new-account? encryption-key handler))
+
+(re-frame/reg-fx
+  :data-store/base-tx
+  (fn [transactions]
+    (async/go (async/>! data-source/realm-queue
+                        (fn []
```

Comment:
this function could be extracted (as it is used twice)

Reply:
Good point